packages/components/src/theme/ComposerTheme/styles.ts

```diff
@@ -180,23 +180,28 @@ export const textStyles = {
 export const inputStyles = {
   large: {
     borderRadius: '0.4rem',
-    px: '0.8rem',
+    ps: '1rem',
+    pe: '1rem',
+    p: '0.75rem 1rem',
+    px: '1rem',
```

Comment:
For some unknown reason... this is needed. If we just set `p` or `px` the `input` ignores that and use a `ps`